aviatrix/resource_aviatrix_firewall_instance.go

```diff
@@ -199,6 +199,14 @@ func resourceAviatrixFirewallInstance() *schema.Resource {
 				Computed:    true,
 				Description: "Management Public IP.",
 			},
+			"tags": {
+				Type:        schema.TypeMap,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Optional:    true,
+				Default:     nil,
+				ForceNew:    true,
```

Comment:
Is it really ForceNew? Interesting, I would think tags would be mutable.

Reply:
Firewall instance has no update method. The API does not support updating/editing firewall instances and the UI doesn't support it either.